.github/workflows/release_deb.yml

```diff
@@ -0,0 +1,118 @@
+name: "Release DEB package"
+
+on:
+  push:
```

Comment:
Release on push? It should be in `release.yml`.

Reply:
Yes, It was for debugging. Now all in the main Release workflow